web-console/Storage/OCS/OCS _Installation_Workflow.md

```diff
@@ -0,0 +1,98 @@
+# OCS Install Workflow
+
+### Operator Hub
+
+There will be 2 OCS operator entries in Operator Hub covering each of the following modes. 
+* Converged Mode -- Local Cluster / Hyperconverged 
+* Independent Mode -- External Cluster (Ceph Cluster is deployed externally)
+
+### Converged Mode Design
+
+Operator Hub
+![Operator Hub operators view](img/OCS-operator-view.png)
+
+Installation
+![Operator Hub operators view](img/OCS-Install-step1.png)
+
+![Operator Hub operators view](img/OCS-Install-step2.png)
```

Comment:
There should be another field called "Installation Mode" depends on how Operator is defined in `spec.installModes` inside the CSV file. 
<img width="554" alt="installModes__Screen_Shot" src="https://user-images.githubusercontent.com/5903705/60311015-17d3f400-990a-11e9-9238-33281c383ce8.png">



It is for specifying if the Operator will be running in a single NS or being available cluster-wide. Could you help check the CSV file and add this field in the mock?

Reply:
I think we decided not to include this selection because the OCS operator should always be available for all namespaces. right @julienlim?

Reply:
Indeed, there is no reason to do it single NS since its an infrastructure that other apps in other NSs would use ...

Reply:
Sounds like your use case will be the same as "App Dynamics" one. I think my point here is you don't need to do any customized UI design here. As long as you set the `spec.InstallMode` in CSV file as to how you stated, the "Create Subscription View" UI should look exactly like this screenshot from "App Dynamics" Operator:
![appDynamics-installMode@2x](https://user-images.githubusercontent.com/5903705/60470574-ca130080-9c15-11e9-8e7f-c35aefe20fa9.png)